src/virtual_machine_start.rs

```diff
@@ -87,12 +76,12 @@ fn print_instructions_for_interactive_console() {
     println!("<an instruction in binary> to instantly execute that instruction");
 }
 
-pub fn execute_vm_in_interactive_mode() -> Result<(), String> {
+pub fn execute_vm_in_interactive_mode() -> Result<(), Error> {
     let program_counter_start = 0x3000;
```

Comment:
You can define a `PC_START` constant for that address.

Reply:
You are right, just moved it to the `constants.rs` file